android/app/src/main/java/app/organicmaps/bookmarks/data/Metadata.java

```diff
@@ -60,7 +60,8 @@ public enum MetadataType
     FMD_JUNCTION_REF(39),
     FMD_BUILDING_MIN_LEVEL(40),
     FMD_WIKIMEDIA_COMMONS(41),
-    FMD_CAPACITY(42);
+    FMD_CAPACITY(42),
+    FMD_WHEELCHAIR(43);
```

Comment:
This approach is ok I think. Just to add a comment here: The value is runtime only, but actual data is taken from the classifier types.

I suppose we will remove it from the classifier someday and add it to the metadata.